packages/database/src/schema.prisma

```diff
@@ -18,3 +18,16 @@ datasource db {
 // this is because these IDs may be used in URLs and it makes them harder to guess
 // while we don't rely on that for security, it adds an extra layer
 // not everything needs this, but easier to make them all consistent and the increase in size (vs int) is negligible
+
+model Otp {
+  id        String   @id @default(uuid())
+  email     String
+  code      String // 5-letter OTP
+  expiresAt DateTime
+  used      Boolean  @default(false)
+  attempts  Int      @default(0)
+  createdAt DateTime @default(now())
+  updatedAt DateTime @updatedAt
```

Comment:
So for this table I think we need only the following
id
email (UNIQUE)
hashed_otp_code
createdAt
expiresAt
attempts

so you can remove updatedAt and used.
We can delete the entry in the db once verified and I think it is important to keep the email column as unique so that there is ever only one row per email.